dtls.c

```diff
@@ -536,6 +536,34 @@ dtls_set_handshake_header(uint8 type,
   return buf;
 }
 
+static const dtls_cipher_t default_cipher_suites[] =
+#ifdef DTLS_DEFAULT_CIPHER_SUITES
+		DTLS_DEFAULT_CIPHER_SUITES ;
+#else
+{
+#ifdef DTLS_ECC
+    TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8,
+    TLS_ECDHE_ECDSA_WITH_AES_128_CCM,
+#endif /* DTLS_ECC */
+#ifdef DTLS_PSK
+    TLS_PSK_WITH_AES_128_CCM_8,
+    TLS_PSK_WITH_AES_128_CCM,
+#endif /* DTLS_PSK */
+    /* TLS_NULL_WITH_NULL_NULL must always be the last entry as it
+     * indicates the stop marker for the traversal of this table. */
+    TLS_NULL_WITH_NULL_NULL
+};
+#endif
+
+static inline uint8_t contains_cipher_suite(const dtls_cipher_t* cipher_suites, const dtls_cipher_t cipher_suite)
```

Comment:
To follow the coding conventions, this should be:
```
static inline uint8_t
contains_cipher_suite(...args...) {
```

(The function name starts on its own line, the opening brace is on the same line as the last formal parameter.)

Reply:
Is it possible to shift this to the code cleanup?
I see many functions, which are coded in the same way.